src/main/resources/openapi.yml

```diff
@@ -315,7 +315,7 @@ components:
         status:
           type: string
         currentTurn:
-          type: string
+          type: array
```

Comment:
An array of what? Don't forget that you need to describe also items that array contains